packages/react-components/react-charts-preview/src/components/Legends/OverflowMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  MenuPopover,
+  MenuTrigger,
+  useOverflowMenu,
+} from '../../../../react-components/src/index';
+
+export const OverflowMenu: React.FC<{ itemIds: string[]; title: string; items: JSX.Element[] }> = ({
+  itemIds,
+  title,
+  items,
+}) => {
+  const { ref, overflowCount, isOverflowing } = useOverflowMenu<HTMLButtonElement>();
+  let displayLabel = title;
+  displayLabel = title === '' ? `+${overflowCount} items` : `+${overflowCount} ${title}`;
```

Comment:
have you tested this control on RTL

Reply:
yes, its working in RTL
![image](https://github.com/user-attachments/assets/4389e893-2b84-49d1-9fc0-09d32a99f2c9)